### ISSUE TITLE
Animation problem with `userChrome.tab.bottom_rounded_corner`

### ISSUE THREAD

**Issue by BPower0036** [black7375/Firefox-UI-Fix#559]
### Describe the bug

There is a problem with the animation of the bottom_rounded_corners.
When you select r hover a tab you see first the corners appearing and then the background of the tab.
If I disable the following rule, there is no problem anymore:
```
button,
toolbarbutton,
stack,
vbox,
.toolbarbutton-icon,
#tabs-newtab-button > .toolbarbutton-icon {
  transition: background-color 1s var(--animation-easing-function) !important;
}
button:hover,
toolbarbutton:hover,
stack:hover,
vbox:hover,
.toolbarbutton-icon:hover,
#tabs-newtab-button:hover > .toolbarbutton-icon {
  transition: background-color 0.25s var(--animation-easing-function) !important;
}
```

You can see it very good when `userChrome.tab.bottom_rounded_corner_edge` of `chrome` is enabled

### Expected behavior

The animation is also for the tab_bottom_corners

### Screenshots

I was able to capture a screenshot midway:
![afbeelding](https://user-images.githubusercontent.com/80090789/210317862-fefebff7-e30d-40e2-b362-44bb266318fc.png)

### OS

Windows 10

### Firefox Version

108.0.1

### Distribution

- [X] [Original Lepton](https://github.com/black7375/Firefox-UI-Fix)
- [ ] [Lepton's photon style](https://github.com/black7375/Firefox-UI-Fix/tree/photon-style)
- [ ] [Lepton's proton style](https://github.com/black7375/Firefox-UI-Fix/tree/proton-style)

### Theme

- [ ] Light
- [ ] Dark
- [ ] Alpenglow (System: Light)
- [ ] Alpenglow (System: Dark)
- [X] System Default
- [ ] Others: (e.g. [Photon Colors](https://addons.mozilla.org/en-US/firefox/addon/photon-colors/), [Google Chrome Dark](https://addons.mozilla.org/en-US/firefox/addon/google-chrome-dark/), [Fairytale Of Nature](https://addons.mozilla.org/en-US/firefox/addon/fairytale-of-nature/))

### `user.js` setup

<details><summary><strong><code>user.js</code> setup</strong></summary><br>

```javascript
userChrome.decoration.animate
userChrome.tab.connect_to_window
userChrome.tab.dynamic_separator
userChrome.tab.box_shadow
userChrome.tab.bottom_rounded_corner
userChrome.tab.bottom_rounded_corner.all
userChrome.tab.bottom_rounded_corner.edge
```
</details>

**Comment by patrik-krajcovic2** [comment]
So you think that animation is not filled within the tab width, but it is cropped?

**Comment by patrik-krajcovic2** [comment]
Well, if I am right, then the australis is affected too (Because I have it enabled).

**Comment by BPower0036** [comment]
Yes, australis is effected to. That animation is within the tab width. But the corners are put on top of it and are not animated. So, at least I,  I see a second difference.

**Comment by Patricek0056** [comment]
I think that this would be difficult to fix, but maybe I am wrong.
